Feature recent works

lewisjn-arch opened the pull request:
This PR has the following:

- [ ] I have added the recent works feature and customized.
- [ ] I have added the about me section.
- [ ] I have styled both sections in CSS.

lewisjn-arch committed:
add my recent works section

lewisjn-arch committed:
update my recent works section with personalisation

lewisjn-arch committed:
style my recent works section

lewisjn-arch committed:
add about me section

lewisjn-arch committed:
style about me section and recent works

lewisjn-arch committed:
add about me section illustration

lewisjn-arch committed:
fix linter errors

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            <h3>Architecture</h3>
+        </div>
+        <div class="feature-project_description_container">
+            <h4 class="feature-project_description">A daily selection of privately personalized reads; no 
+            accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</h4>
```
Kindly, I suggest you use the `p` tag instead of the `h4` to follow HTML best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            <li class="feature-skill_container"><h4 class="feature-skill">Ruby</h4></li>
+        </ul>
+            <button class="feature-project_link">
+                <h4 class="feature-link_text">See project</h4>
```
Kindly, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `button` tag I suggest you use a `span` instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            when an unknown printer took a standard dummy text.</h4>
+        </div>
+        <ul class="feature-project_skills">
+            <li class="feature-skill_container"><h4 class="feature-skill">CSS</h4></li>
+            <li class="feature-skill_container"><h4 class="feature-skill">HTML</h4></li>
+            <li class="feature-skill_container"><h4 class="feature-skill">Bootstrap</h4></li>
+            <li class="feature-skill_container"><h4 class="feature-skill">Ruby</h4></li>
```
Kindly, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `li` tag I suggest you use a `span` or a `p` tag instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <h3>E-commerce Website</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
```
Kindly, I suggest you use a `p` tag instead of a `h4` tag for the description, please, note that this is for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
```
As mentioned above, kindly, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `li` tag I suggest you use a `span` or a `p` tag instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <h3>University Portal</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
```
As I mentioned above, kindly, I suggest you use a `p` tag instead of a `h4` tag for the description, please, note that this is for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
```
As mentioned above, kindly, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `li` tag I suggest you use a `span` or a `p` tag instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <h3>Company Website</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
```
As I mentioned above, kindly, I suggest you use a `p` tag instead of a `h4` tag for the description, please, note that this is for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
```
As mentioned above, kindly, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `li` tag I suggest you use a `span` or a `p` tag instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <h3>Business Website</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
```
As I mentioned above, kindly, I suggest you use a `p` tag instead of a `h4` tag for the description, please, note that this is for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
```
As mentioned above, kindly, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `li` tag I suggest you use a `span` or a `p` tag instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <h3>Bank Website</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
```
As I mentioned above, kindly, I suggest you use a `p` tag instead of a `h4` tag for the description, please, note that this is for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
```
As mentioned above, kindly, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `li` tag I suggest you use a `span` or a `p` tag instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <h3>Trading Website</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
```
As I mentioned above, kindly, I suggest you use a `p` tag instead of a `h4` tag for the description, please, note that this is for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
```
As mentioned above, kindly, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `li` tag I suggest you use a `span` or a `p` tag instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
```
As mentioned above, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `button` tag I suggest you use a `span` instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
```
As mentioned above, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `button` tag I suggest you use a `span` instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
```
As mentioned above, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `button` tag I suggest you use a `span` instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
```
As mentioned above, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `button` tag I suggest you use a `span` instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
```
As mentioned above, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `button` tag I suggest you use a `span` instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
```
As mentioned above, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `button` tag I suggest you use a `span` instead of a `h4` for good semantics and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            <h1>About me</h1>
+        </div>
+        <div class="bio">
+            <h4>Hello I'm a software developer! I can help you build a product , feature or website 
+            Look through some of my work and experience! If you like what you see and have a 
+            project you need coded, don't hesitate to contact me.</h4>
```
As mentioned above, I suggest you use the heading tag elements only for a heading and sub-heading title. please, replace the `h4` tag with a `p` tag instead for good semantics reasons and best practices. 😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            <hr>
+        </div>
+        <div class="knowledge">
+            <div class="competent">
+                <div class="category_style">
+                    <div class="bullet_1"></div>
+                    <h3 class="category">Language</h3>
+                </div>
+                <h4 class="competency">Javascript</h4>
+                <h4 class="competency">Ruby</h4>
+                <h4 class="competency">HTML</h4>
+                <h4 class="competency">CSS</h4>
+            </div>
```
- Kindly, make sure you use the `ul` tag to list the elements on this block instead of using the `div` tag, and note that each should be inside a `li` tag.😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <h4 class="competency">HTML</h4>
+                <h4 class="competency">CSS</h4>
+            </div>
+            <div class="competent">
+                <div class="category_style">
+                    <div class="bullet_2"></div>
+                    <h3 class="category">Frameworks</h3>
+                </div>
+                <h4 class="competency">React.js</h4>
+                <h4 class="competency">Ruby on Rails</h4>
+                <h4 class="competency">RSpec</h4>
+                <h4 class="competency">Capybara</h4>
+            </div>
```
- Kindly, make sure you use the `ul` tag to list the elements on this block instead of using the `div` tag, and note that each should be inside a `li` tag.😄

stanleySimeon commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <h4 class="competency">RSpec</h4>
+                <h4 class="competency">Capybara</h4>
+            </div>
+            <div class="competent">
+                <div class="category_style">
+                    <div class="bullet_3"></div>
+                    <h3 class="category">Skills</h3>
+                </div>
+                <h4 class="competency">Database Management</h4>
+                <h4 class="competency">Version Control</h4>
+                <h4 class="competency">CLI</h4>
+                <h4 class="competency">API Design</h4>
+                <h4 class="competency">Web Development</h4>
+            </div>
```
- Kindly, make sure you use the `ul` tag to list the elements on this block instead of using the `div` tag, and note that each should be inside a `li` tag.😄

According to the Figma design, I suggest you fix some margins and padding to make your project match the Figma Design.

> ## Your Design
>
><img width="372" alt="Screenshot 2023-06-03 at 5 52 15 PM" src="https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/91237525/d827ec28-30b2-4d87-8046-4cbe7634ca7c">
>
> ## The Figma Design
>
><img width="465" alt="Screenshot 2023-06-03 at 5 53 13 PM" src="https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/91237525/7ca71f6e-068a-40d8-8618-d5c615eba6f7">

lewisjn-arch replied:
@stanleySimeon  Thank you so much for your review, let me get to it right now as I was unwell with a flu hence I am a bit late with the project.

stanleySimeon reviewed (changes requested):
# Hi @lewisjn-arch, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media2.giphy.com/media/vQGBDTkhtIunaJYmRF/giphy-downsized.gif?cid=29caca75sf3uq9y4qgulk5ybbojwg94b4zo6temfzeel417x&ep=v1_gifs_search&rid=giphy-downsized.gif&ct=g
)

## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- adding your live demo link

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lewisjn-arch committed:
replace header tags with correct tags, correct ul tags

lewisjn-arch committed:
correct margins of about me section to match design

lewisjn-arch committed:
fix linter errors

lewisjn-arch commented:
This second code review request contains the following:

- [ ] I replaced header tags with correct semantic tags.
- [ ] I replaced div tags with correct ul tags.
- [ ] I fixed margins on the skills section to match the figma design.

AbayomiOlaoye commented on index.html:
```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+    </section>
+    <section class="about_me">
+        <div class="bio_header">
+            <h1>About me</h1>
```
- [ ] [OPTIONAL] Kindly use another `hx` value for this heading. As best practice, an `h1` should not be more than one in a document. This confuses the major focus or central theme of the page - and this can have a negative effect on the SEO of the page

lewisjn-arch replied:
@AbayomiOlaoye Thank you so much, I will get to it right away.

AbayomiOlaoye reviewed (approved):
## STATUS: APPROVED :green_circle:  

Hi @lewisjn-arch   :wave: :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![applause](https://user-images.githubusercontent.com/99715640/213818870-485b6c36-1d87-4b93-9a0c-ce2ae86d9a9d.gif)

## Highlights
- All changes made :white_check_mark: ✅ 
- Good use of GitfHub flow ✅ 
- Professional description ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Ibizugbe reviewed (commented):
Hi @lewisjn-arch 👋👋👋, 

![StandingOvationGIF (6)](https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/64643995/4f801427-8712-49c6-83e7-23f369fa57e8)


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅

- [x] Core requirements of the project have been met👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
